rrmngmnt/host.py

```diff
@@ -412,24 +412,3 @@ def create_script(
         except errors.CommandExecutionFailure:
             return ""
         return dst
-
-    def is_connective(self, tcp_timeout=20.0):
```

Comment:
I would like to avoid to remove this method right now.
please use warnings module to mark it as deprecated and we will remove it in next release cycle.

you can see how I did it here: https://github.com/rhevm-qe-automation/python-rrmngmnt/pull/33/files#diff-6f59a17db9ff2c03058e71e00b2c56baR400